src/index.ts

```diff
@@ -1,5 +1,7 @@
 /// <reference types="./ffprobe" />
-import { path as ffprobePath } from '@ffprobe-installer/ffprobe'
+// eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+// @ts-ignore
```

Comment:
Why is this ignore neeed?

Reply:
@Sumolari Well… I don’t know quite know TypeScript and if I don’t ts-ignore that line it doesn’t work. Any suggestions?

Reply:
I think you are missing changing `'@ffprobe-installer/ffprobe'` by `'node-ffprobe-installer'` in `src/ffprobe.d.ts`

Reply:
Done! Ready to merged.